include/em4x70.h

```diff
@@ -0,0 +1,18 @@
+//-----------------------------------------------------------------------------
+// Copyright (C) 2020 sirloins
+//
+// This code is licensed to you under the terms of the GNU GPL, version 2 or,
+// at your option, any later version. See the LICENSE.txt file for the text of
+// the license.
+//-----------------------------------------------------------------------------
+// Low frequency EM4x70 structs
+//-----------------------------------------------------------------------------
+
+#ifndef EM4X70_H__
+#define EM4X70_H__
+
+typedef struct {
+    bool parity;
+} PACKED em4x70_data_t;
```

Comment:
packed struct with a bool??

Reply:
I sort of copied the em4x50_data_t struct, I removed the items I wasn't using yet. I have added more items for the next pull request that includes writing support.

I'll remove the PACKED attribute in this push.

Reply:
It just didn't make any sense to have an struct with a bool inside.  So its for next PRs  ok.